.changeset/@graphql-hive_gateway-481-dependencies.md

```diff
@@ -0,0 +1,7 @@
+---
+'@graphql-hive/gateway': patch
```

Comment:
I think the "eventual" changeset here will be missing some details about the upgrades? 
@enisdenjo can you please add more info to the main release changeset? I know the main library is not affected a lot here, but I think we should communicate it in the changeset of the gw itself and not only the internal pkgs?

Reply:
@dotansimha have you seen the other changesets?

Reply:
Yeah, the reason I asked it is because the rest of the changesets are for internal packages, and I wonder if we need one specifically to the root package, since this is what users usually look for?

Reply:
we can add `'@graphql-hive/gateway': patch` to one of the other changesets? (which are explained well! thank you!)

Reply:
@dotansimha ok I added some where I thought it's relevant.